package.json

```diff
@@ -13,10 +13,20 @@
     "chai": "^4.2.0"
   },
   "devDependencies": {
-    "cypress": "^4.5.0"
+    "cypress": "^4.5.0",
+    "cypress-multi-reporters": "^1.4.0",
+    "mocha": "^8.0.1",
+    "mochawesome": "^6.1.1",
+    "mochawesome-merge": "^4.1.0",
+    "mochawesome-report-generator": "^5.1.0"
   },
     "scripts": {
-      "chrome:headless": "cypress run --headless --browser chrome",
-      "chrome:head": "cypress run --browser chrome"
+      "clean:reports": "rm -R -f cypress/reports && mkdir cypress/reports",
+      "pretest": "npm run clean:reports",
+      "scripts": "cypress run",
```

Comment:
You can create 1 command which will include 2 commands in one, e.g.:
`"cypress:tests": "deleteNode && runTests"`

Reply:
Would it be ok to add it to `"test" : "**npm run pretest** || npm run scripts || npm run posttest"`?